### PR TITLE
Liquid & Liquid Assets Support

### PR DESCRIPTION
**Pull request by Kukks**
~Requires https://github.com/dgarage/NBXplorer/pull/198 first~

This PR introduces support for the Liquid Network and its assets. This currently supports LBTC(the main asset, 1:1 with BTC) and Liquid Tether (USDT). 

Since Liquid seems to break the PSBT support we have, the wallet section for them is disabled.

### CODE REVIEW

File: BTCPayServer.Tests/BTCPayServerTester.cs [hunk 1]
```diff
@@ -109,13 +110,15 @@ public async Task StartAsync()
                 config.AppendLine($"bind=0.0.0.0");
             }
             config.AppendLine($"port={Port}");
-            config.AppendLine($"chains=btc,ltc");
+            config.AppendLine($"chains=btc,ltc,lbtc,usdt");
```
Comment (NicolasDorier) [inline BTCPayServer.Tests/BTCPayServerTester.cs]: add lbtc and usdt only for tests actually testing liquid.

**Comment by NicolasDorier** [comment]
Need rebase. Please rebase and force push instead of merging.

**Comment by Kukks** [comment]
> 
> 
> Need rebase. Please rebase and force push instead of merging.

rebased

File: BTCPayServer.Tests/BTCPayServerTester.cs (hunk 1 above)
Comment (Kukks) [reply BTCPayServer.Tests/BTCPayServerTester.cs]: done

File: BTCPayServer.Tests/BTCPayServerTester.cs [hunk 2]
```diff
@@ -93,7 +94,7 @@ public TestDatabases TestDatabase
 
         public bool MockRates { get; set; } = true;
 
-        public async Task StartAsync()
+        public async Task StartAsync(bool addLiquid = false)
```
Comment (NicolasDorier) [inline BTCPayServer.Tests/BTCPayServerTester.cs]: no parameter, just use a property that your test set before running Start. (will do the same for litecoin later)

Comment (Kukks) [reply BTCPayServer.Tests/BTCPayServerTester.cs]: fixed

**Pull request merged**